### PR TITLE
Testing2

### LINKED ISSUE #49: Refactor Gamepage Test

### PR DESCRIPTION
**Pull request by Chriscastanuela**
## Description
* Creates and passes new tests for `Gamepage.js`, `Lobby`, and `WordSelector`

## Where To Start
* `Gamepage.js`, `Lobby`, and `WordSelector`

## Affected areas of application
* `Gamepage.js`, `Lobby`, and `WordSelector`

## Screenshots (if applicable):
Screen Shot 2020-11-02 at 4 35 29 PM

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] Test A - Jest

## Relevant Tickets
* Closes #49 

## Tags
@holladayian 
@GreyMatteOr

### CODE REVIEW

File: src/Gamepage/Gamepage.test.js [hunk 1]
```diff
@@ -99,18 +99,20 @@ describe('Gamepage', () => {
...
+          }
         }
-      }
       />);
-      expect(screen.getByText('cuts')).toBeInTheDocument();
-      expect(screen.getByText('butts')).toBeInTheDocument();
-      expect(screen.getByText('coconuts')).toBeInTheDocument();
+      expect(screen.getByText('one')).toBeInTheDocument();
+      expect(screen.getByText('two')).toBeInTheDocument();
+      expect(screen.getByText('three')).toBeInTheDocument();
```
Comment (holladayian) [inline src/Gamepage/Gamepage.test.js:115]: Is this really what your heart desires?

**Review by holladayian — APPROVED** [review]

File: src/Lobby/Lobby.test.js [hunk 2]
```diff
@@ -8,9 +8,12 @@ describe('Lobby', () => {
...
-    render(<Lobby designateRole={mockFn}/>);
+    render(<Lobby rooms={['chris', 'ian', 'matt']} designateRole={mockFn}/>);
 
-    expect(screen.getByText('Generator')).toBeInTheDocument();
-    expect(screen.getByText('Guesser')).toBeInTheDocument();
+    expect(screen.getByText('chris')).toBeInTheDocument();
+    expect(screen.getByText('ian')).toBeInTheDocument();
+    expect(screen.getByText('matt')).toBeInTheDocument();
+    expect(screen.getByText('Available Rooms')).toBeInTheDocument();
+    expect(screen.getByPlaceholderText('Room ID')).toBeInTheDocument();
```
Comment (holladayian) [inline src/Lobby/Lobby.test.js:17]: Nice! I think we can test for the buttons too? I'm not too partial about it though

**Review by holladayian — APPROVED** [review]

File: src/Gamepage/Gamepage.test.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ describe('Gamepage', () => {
   describe('Render method / Headers', () => {
     it('3. should render the `Hints` header', () => {
 
-      render(<Gamepage display={['d', '_', '_', 'o', '_', 'a', 'u', 'r']} attempts={['cuts', 'butts', 'coconuts']} isGenerator={null}
+      render(<Gamepage display={['d', '_', '_', 'o', '_', 'a', 'u', 'r']} attempts={['one', 'two', 'three']} isGenerator={null}
```
Comment (GreyMatteOr) [inline src/Gamepage/Gamepage.test.js:42]: These all look like prime `beforeEach` fodder.

**Review by GreyMatteOr — APPROVED** [review]
TIght. I think you really have a knack for testing!

**Pull request merged**